control/autopilot/include/autopilot/autopilot_inl.h

```diff
@@ -1085,13 +1141,20 @@ quadrotor_common::ControlCommand AutoPilot::velocityControl(
 
   time_last_velocity_command_handled_ = ros::Time::now();
 
+
+  reference_trajectory_ = quadrotor_common::Trajectory();
+  reference_trajectory_.trajectory_type = 
+    quadrotor_common::Trajectory::TrajectoryType::GENERAL;
```

Comment:
quick question, how about a constructor with the type as argument for the class `quadrotor_common::Trajectory()` ? to avoid code duplication. We could even have a constructor with one point as argument and automatically defined it as a general trajectory. It would make sense since we change the interface completely (i mean from point to trajectory in the run command...)

Reply:
Agree! Will do it tomorrow!

Reply:
also agree ;)

Reply:
done in https://github.com/uzh-rpg/rpg_quadrotor_common/pull/11